bihire/property_pro_lite
# setting up the server

#### What does this PR do?
- servers as an entry point to the server

#### Description of Task to be completed
- A user should be able to hit url and then be redirected to appropriate endpoints

#### How should this be manually tested?
- clone the repo and switch to the branch ft-signup-123456
- start the server
- use the url `http://localhost:8081/ ` to open the welcome page page
- you should be successfully redirected to index.html

#### Any background context
- N/A

#### What are the relevant pivotal tracker stories?
[#167188158](http://pivotaltracker.com)
